### PR TITLE
Add support for `resources_by_asset_folder` for Admin API

### PR DESCRIPTION
**Pull request by tommyg-cld**
#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [x] New feature
- [ ] Bug fix
- [ ] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

File: lib/cloudinary/api.rb [hunk 1]
```diff
@@ -189,6 +189,24 @@ def self.resources_by_asset_ids(asset_ids, options={})
     call_api(:get, uri, params, options)
   end
 
+  # Returns all assets stored directly in a specified asset folder, regardless of the public ID paths of those assets.
+  #
+  # @param [Object] asset_ids The requested asset IDs.
```
Comment (const-cloudinary) [inline lib/cloudinary/api.rb:194]: The first parameter is actually `asset_folder` and it is a string

**Review by const-cloudinary — CHANGES_REQUESTED** [review]

File: lib/cloudinary/api.rb (hunk 1 above)
Comment (tommyg-cld) [reply lib/cloudinary/api.rb]: This has been updated.

File: lib/cloudinary/api.rb [hunk 2]
```diff
@@ -189,6 +189,24 @@ def self.resources_by_asset_ids(asset_ids, options={})
     call_api(:get, uri, params, options)
   end
 
+  # Returns all assets stored directly in a specified asset folder, regardless of the public ID paths of those assets.
+  #
+  # @param [String] asset_ids The requested asset IDs.
```
Comment (const-cloudinary) [inline lib/cloudinary/api.rb:194]:
```suggestion
  # @param [String] asset_folder The requested asset IDs.
```

Comment (const-cloudinary) [inline lib/cloudinary/api.rb:194]: The parameter name and description are still inaccurate

Comment (tommyg-cld) [reply lib/cloudinary/api.rb]: I've updated them as well.

**Pull request merged**